.github/workflows/main.yaml

```diff
@@ -6,6 +6,10 @@ on:
   push:
     branches: [main]
 
+  schedule:
```

Comment:
I think we should run just e2e tests and we should run it on the 'real' deployed opscenter version

Reply:
well, that's what I suggested this morning. Let's talk before I do anything else. Thanks!

Reply:
then we both agree, lets do it!